.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2.1
+
+jobs:
+  linux-aarch64-wheels:
+    working_directory: ~/linux-aarch64-wheels
+    machine:
+      image: ubuntu-2004:2022.04.1
+      docker_layer_caching: true
+    # resource_class is what tells CircleCI to use an ARM worker for native arm builds
+    # https://circleci.com/product/features/resource-classes/
+    resource_class: arm.medium
+    environment:
+      CIBUILDWHEEL: 1
+      CIBW_BUILD: "cp*-manylinux_aarch64"
+      CIBW_ENVIRONMENT_PASS_LINUX: "LD_LIBRARY_PATH"
+      # VCPKG_INSTALL: "$VCPKG_INSTALLATION_ROOT/installed/arm64-linux-dynamic-release"
+    steps:
+      - checkout
+      - run:
+          name: Build docker image with GDAL install
+          command: docker build -f ci/manylinux_2_28_aarch64-vcpkg-gdal.Dockerfile -t manylinux-aarch64-vcpkg-gdal:latest .
+      - run:
+          name: Build the Linux aarch64 wheels.
+          command: |
+            python3 -m pip install --user cibuildwheel==2.12.1
+            python3 -m cibuildwheel --output-dir wheelhouse
+      - run:
+          name: Test the wheels
+          command: |
+            python3 -m pip install -r ci/requirements-wheel-test.txt
+            python3 -m pip install --no-deps geopandas
+            python3 -m pip install --pre --find-links wheelhouse pyogrio
+            python3 -m pip list
+            cd ..
+            python3 -c "import pyogrio; print(f'GDAL version: {pyogrio.__gdal_version__}\nGEOS version: {pyogrio.__gdal_geos_version__}')"
+            python3 -m pytest --pyargs pyogrio.tests -v
```

Comment:
That is probably OK for now; we don't tend to use a lot of Python version-specific features, so this is mostly a test that the aarch64 build itself produced valid wheels rather than needing to do exhaustive testing of every wheel.  We can revisit if we get reports that some of the aarch64 wheels do not work properly.